lib/src/record_replay/manifest.dart

```diff
@@ -47,25 +45,28 @@ class Manifest {
   /// [toJson]), a [FormatException] will be thrown.
   factory Manifest.fromJson(String json) {
     List<Map<String, dynamic>> decoded =
-        new JsonDecoder().convert(json).cast<Map<String, dynamic>>();
-    Manifest manifest = new Manifest();
+        (JsonDecoder().convert(json) as List<dynamic>)
+            .cast<Map<String, dynamic>>();
+    Manifest manifest = Manifest();
```

Comment:
[optional] If you're doing cleanup anyway, consider removing some of the very clearly redundant local variable types.

Reply:
always_specify_types 🙃